test/cql-pytest/test_bad_grammar.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests for the parser, focusing on rejected grammar
+#############################################################################
+
+import pytest
+from util import new_test_table
+from cassandra.protocol import InvalidRequest, SyntaxException
+
+
+# A collection of tests for grammar that should be rejected, but wasn't at
+# some point in the past.
+
+# table1 has some pre-set data which the tests below SELECT on (the tests
+# shouldn't write to it).
```

Comment:
You don't need the data in this test, it's completely irrelevant. So please remove this comment, and all the prepare and execute statements in the table1 fixture. Only need:
```python
@pytest.fixture(scope="module")
def table1(cql, test_keyspace):
    with new_test_table(cql, test_keyspace, "p int, c1 int, c2 int, v int, PRIMARY KEY (p, c1, c2)") as table:
        yield table
```